Aikido.Zen.Core/Models/Ip/Blocklist.cs

```diff
@@ -137,17 +181,92 @@ public bool IsIPAllowed(string ip, string endpoint)
                     return true; // Allow invalid IPs by default
                 }
 
-                if (!_allowedSubnets.TryGetValue(endpoint, out var trie))
+                var parts = endpoint.Split('|');
```

Comment:
we should be able to access these fields "method" and "route" directly from endpoint without string parsing right?